mmv1/products/cloudbuild/api.yaml

```diff
@@ -83,8 +83,83 @@ objects:
+          Path, from the source root, to a file whose contents is used for the template. 
+          Either a filename or build template must be provided. Set this only when using trigger_template or github.
+          When using Pub/Sub, Webhook or Manual set the file name using git_file_source instead.
+      - !ruby/object:Api::Type::String
+        name: 'filter'
+        description: |
+          A Common Expression Language string. Used only with Pub/Sub and Webhook.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'gitFileSource'
+        exactly_one_of:
+          - filename
+          - git_file_source
+          - build
+        description: |
+          The file source describing the local or remote Build template.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'path'
+            required: true
+            description: |
+              The path of the file, with the repo root as the root of the path.
+          - !ruby/object:Api::Type::String
+            name: 'uri'
+            description: |
+              The URI of the repo (optional). If unspecified, the repo from which the trigger 
+              invocation originated is assumed to be the repo from which to read the specified path.
+          - !ruby/object:Api::Type::Enum
+            name: 'repoType'
+            required: true
+            description: |
+              The type of the repo, since it may not be explicit from the repo field (e.g from a URL).
+            values:
+                  - :UNKNOWN
+                  - :CLOUD_SOURCE_REPOSITORIES
+                  - :GITHUB
+          - !ruby/object:Api::Type::String
+            name: 'revision'
+            description: |
+              The branch, tag, arbitrary ref, or SHA version of the repo to use when resolving the 
+              filename (optional). This field respects the same syntax/resolution as described here: https://git-scm.com/docs/gitrevisions 
+              If unspecified, the revision from which the trigger invocation originated is assumed to be the revision from which to read the specified path.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'sourceToBuild'
+        description: |
+          The repo and ref of the repository from which to build. 
+          This field is used only for those triggers that do not respond to SCM events. 
+          Triggers that respond to such events build source at whatever commit caused the event. 
+          This field is currently only used by Webhook, Pub/Sub, Manual, and Cron triggers.
+        at_least_one_of:
+          - trigger_template
+          - github
+          - pubsub_config
+          - webhook_config
+          - source_to_build
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'uri'
+            required: true
+            description: |
+              The URI of the repo (optional). If unspecified, the repo from which 
```

Comment:
Is this the correct description? It looks like it's marked required, but described as optional.

Reply:
Thanks for reviewing @megan07. I have updated the description.